src/app/base/components/node/HardwareCard/_index.scss

```diff
@@ -0,0 +1,26 @@
+@mixin HardwareCard {
+    .hardware-card {
+        padding: 0.5rem 1rem 1rem 1rem;
+        
+        .p-list__item + .p-list__item {
```

Comment:
If you pass `p-list--divided` to the `LabelledList` components then you can remove this.

Reply:
Oh sweet! That's way neater